src/System.Threading.Channels/ref/Configurations.props

```diff
@@ -1,8 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
+      netstandard1.3;
       netstandard;
+    </PackageConfigurations>
+    <BuildConfigurations>
```

Comment:
You don't need to split this into package and build configurations given they are going to be the same thing.